File: src/main/java/me/kuku/yuq/logic/QQMailLogic.java
package me.kuku.yuq.logic;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.QQLoginEntity;
import me.kuku.yuq.pojo.Result;

import java.io.IOException;
import java.util.List;
import java.util.Map;

@AutoBind
public interface QQMailLogic {
    Result<List<Map<String, String>>> getFile(QQLoginEntity qqLoginEntity) throws IOException;
    String fileRenew(QQLoginEntity qqLoginEntity) throws IOException;
}

File: src/main/kotlin/me/kuku/yuq/logic/QQMailLogic.kt
package me.kuku.yuq.logic

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.QQLoginEntity
import me.kuku.yuq.pojo.CommonResult

@AutoBind
interface QQMailLogic {

    fun getFile(qqLoginEntity: QQLoginEntity): CommonResult<List<Map<String, String>>>

    fun fileRenew(qqLoginEntity: QQLoginEntity): String

}